core/App/utils/helpers.ts

```diff
@@ -28,9 +28,28 @@ export function hashCode(s: string): number {
   return s.split('').reduce((hash, char) => char.charCodeAt(0) + ((hash << 5) - hash), 0)
 }
 
-export function hashToRGBA(i: number) {
-  const colour = (i & 0x00ffffff).toString(16).toUpperCase()
-  return '#' + '00000'.substring(0, 6 - colour.length) + colour
+/**
+ * seeded random number generator between 0-1
```

Comment:
Are you able to put the comment into [JSDoc](https://github.com/jsdoc/jsdoc) format?